### ISSUE TITLE
[Feature Request] Add a button to manually refresh/redownload chapter list

### ISSUE THREAD

**Issue by alkumaish** [Suwayomi/Tachidesk-WebUI#184]
As per the title:  Please add a button to manually refresh the current chapter list. 

This feature already exists in other tachidesk clients. It is useful because sometimes the chapter list does not get updated automatically.

**Comment by Robonau** [comment]
loading the page always attempts a chapter list refresh

**Comment by alkumaish** [comment]
> loading the page always attempts a chapter list refresh
That's the expected behaviour, but I couldn't get the chapter list to update but simply reloading the page. I double checked the source website and it indeed had new chapters; then I just ran sorayomi, updated the chapter list from there and voila: new chapters are now visible on webUI.
So far I have tested this with all comics I have (all from mangakik extension), and it was I got the same result.  I don't follow many, so my ability to test this is limited to what I have.
As of now, I can't tell if this can be repeated with other sources because that's what I only had. So I added few active comics from other sources to test this further.
